lib/TestScriptRunnable.rb

```diff
@@ -116,18 +117,24 @@ def run client = nil
   def execute op
     return unless op
 
+    FHIR.logger.info "[.execute]: #{op.description}"
+
     catch :exit do
       throw :exit, report.fail('noClient') unless client
       throw :exit, report.fail('noRequestType') unless op.type&.code || op.local_method
-
+      
       request_type = REQUEST_TYPES[op.local_method || op.type.code]
       throw :exit, report.skip('notImplemented') unless request_type
 
       request = [request_type, extract_path(op), extract_body(request_type, op), extract_headers(op)]
       request.compact!
 
       begin
-        reply = client.send *request
+        if op.type.code == 'history'
+          reply = client.resource_instance_history(op.class, id_map[op.targetId])
```

Comment:
```suggestion
          reply = client.resource_instance_history(op.class, id_map[op.targetId])
```
I'm going to take a closer look at this while starting the unit tests. The way the request array is created, we shouldn't have to interrogate the operation for its type -- every type should just work. So, if that's not the case, we need to figure out why and how to fix the way 'request' is constructed.

Reply:
I wasn't sure how 'client and.send' and 'request' cover history request, couldn't find an example of code. I agree with you that one method should be abstracted enough to cover all types from a class.